lib/tasks/evm_ansible_runner.rake

```diff
@@ -0,0 +1,35 @@
+require 'awesome_spawn'
+require "vmdb/plugins"
+
+namespace :evm do
+  namespace :ansible_runner do
+    desc "Seed galaxy roles for provider playbooks"
+    task :seed do
+      plugins_with_req_yml = Vmdb::Plugins.select do |plugin|
+        req_yml_path = plugin.root.join('content_tmp', 'ansible', 'requirements.yml')
```

Comment:
Let's encode this into `VMDB::Plugins` a la `VMDB::Plugins::AnsibleContent`

Reply:
:+1: done